### PR TITLE
Ensure Links to Project Resources Are Clear with Media Type Descriptions (Ticket #2090)  

### PR DESCRIPTION
**Pull request by mai-repo**
### Summary:

This PR ensures all supplemental material links are updated with the appropriate media type descriptions for better clarity and accessibility. For example:  
- **How to Dump and Restore PostgreSQL Database? (Article)**  

The following project files were reviewed and updated:  
- **2023 Template With Vite**  
- Blog App  
- Contact App  
- Eventful API Test Page  
- Eventonica (Legacy Version & Updated Version)  
- Final Project  
- Node HTTP Exercise  
- Portfolio  
- Recipe Page  
- Take-Home Problems  
- Advanced Methods Assessment  
- Auth0 Assessment  
- Data Structures and Algorithms Assessment  
- Data Structures and Algorithms Sprint  
- Database Modeling Assignment  
- Full-Stack Assessment  
- Full-Stack Checklist  
- JavaScript & HTML Games  
- Kickball Signups Project  
- Mailing List REST API  
- MERN/PERN Project  
- Minimal API Project  
- MongoDB Assessment  
- PERN Contact List App  
- PERN Game App  
- PERN Weather App  
- React Assessment  
- React Game  
- React Tweeter  
- REST API Project  
- Submitting Work  
- Utils.js  
- Weather App  
- Week 1 Worksheet  
- Week 8 Game README  

### Changes Made:

**1. Link Fixed:**  
   - Fixed the link to the Git file in `minimal-api-project.md`.  

**2. Media Type Descriptions:**  
  - Added media type descriptions (e.g., "Article", "Website") in:  
     - `linked-lists.md`  
     - `queues.md`  
     - `REST-API.md`  
     - `rest-api-project.md`  
     - `week8GameREADME.md`  
   - Improved link clarity in `backend.md` by adding descriptive names.

### Objective:
To ensure that all links to resources are easy to navigate, clear, and provide sufficient context for the user.  

### Next Steps:
- @mx-ruthie, @daaimah123, and @stmcpeters please review the media type descriptions that I added. :)

### CODE REVIEW

**Review by daaimah123 — APPROVED** [review]
# These are spot on changes! 🥳 

@mai-repo  I have approved your changes. Pending reviews from your mentor and a peer...

- Please acknowledge and "resolve" any outstanding comments that you have not interacted with.
- Please let me know if you have any additional incoming changes. If not, go ahead and "Merge pull request".
- Ensure both your related GH issue and PR status have been appropriately updated to "Implemented or Addressed" (this should happen automatically, but if not do so manually).
- Leave a respective comment on your related GH issue closing it.

File: projects/2023TemplateWithVite/README.md [hunk 1]
```diff
@@ -66,5 +66,5 @@ For this template, the name of your db should be `techtonica`.
 
 ⚙️ Links that you could need:
 
-- The instructions for [pg](https://node-postgres.com/apis/pool)
-- Setup [postgres correctly](https://github.com/Techtonica/curriculum/blob/main/databases/installing-postgresql.md)
+- The instructions for [pg](https://node-postgres.com/apis/pool) (Webite)
```
Comment (mx-ruthie) [inline projects/2023TemplateWithVite/README.md:69]: Website (s is missing)

File: projects/recipe-page/phase-1-html-prompt.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ Copy the URL that appears in the search bar -- this is the URL you can  use insi
 `src` attribute of your `img` tag!
 </pre>
 
-[Here's a short video that walks you through the steps given above](https://youtu.be/lTJWBagWE4c).
+[Here's a short nineteen seconds video that walks you through the steps given above](https://youtu.be/lTJWBagWE4c).
```
Comment (mx-ruthie) [inline projects/recipe-page/phase-1-html-prompt.md:113]: nineteen second (with no s)

**Review by mx-ruthie — APPROVED** [review]
marked a few typos, otherwise lgtm

File: projects/recipe-page/phase-1-html-prompt.md (hunk 2 above)
Comment (mai-repo) [reply projects/recipe-page/phase-1-html-prompt.md]: Thanks Ruthie, just made the change here is the commit hash 665efbaf8374dfd5b659ee0e0e0ae5d34fbdd7c6

File: projects/2023TemplateWithVite/README.md (hunk 1 above)
Comment (mai-repo) [reply projects/2023TemplateWithVite/README.md]: Thanks, Ruthie! I have corrected the spelling and the commit hash for this change is 9f223db261e90f9ae1ff69c6ae4b3edf614590cf

**Review by jsnorek — APPROVED** [review]
Great job being so thorough with the links! Looks good 👍

**Pull request merged**